crm_rma_stock_location/product.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Guewen Baconnier
+#    Copyright 2014 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+import openerp.addons.decimal_precision as dp
+from openerp.tools.float_utils import float_round
+
+
+class ProductProduct(models.Model):
+
+    _inherit = 'product.product'
+
+    @api.one
+    @api.depends('rma_qty_available', 'rma_virtual_available')
```

Comment:
This depends is not correct. You should put the source fields that modifies this computation. Leaving store=False, you don't need to put anything.